src/components/date.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
+
+/**
+ * Interface for the DateStrProps
+ */
+interface DateStrProps {
+    date: string,
+    locals: string,
+    options: Intl.DateTimeFormatOptions
+}
+
+/**
+ * Default options for date formatting
+ */
+const DEFAULT_OPTIONS: Intl.DateTimeFormatOptions = {timeZone: 'UTC', month:"long", day:"numeric", year:"numeric"};
+
+/**
+ * DateStr component
+ * @param date - The date string
```

Comment:
Maybe add to the documentation to make it clearer what date, locals, and options are allowed:  `
```
@param date - A string appropriate as the single argument to the Date() constructor
@param locals - An optional string used as the first argument to the Date.toLocalDateString() method (default: 'en-us')
@param options - An optional objected used as the second argument to the Date.toLocalDateString() method
```